package.json

```diff
@@ -5,7 +5,7 @@
   "private": "false",
   "scripts": {
     "test": "hardhat test && forge test --via-ir",
-    "test:foundry": "forge test --via-ir",
+    "test:foundry": "forge test --via-ir -vvv",
```

Comment:
Yeah I think it makes sense in that way because `test:foundry` mainly will be run to debug, since using just `test` implies in not showing the full verbose, which would be run to run both foundry and hardhat tests.

Also a good note about that is that `-vvv` will show verbose for failing tests and `-vvvv` will show full verbose for all tests.

Reply:
I agree with going verbose on failing tests.